lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -113,7 +113,8 @@ describe('CategoryRenderer', () => {
     category.description = prevDesc;
   });
 
-  it('renders audits with debugString as failed', () => {
+  // TODO(phulce): decide what to do about these cases
```

Comment:
updated the comment. this is what's up for debate re: toplevel warnings

currently the test simply doesn't apply anymore since all warnings are stuck into toplevel for display